### ISSUE TITLE
Automatic rejection not handled

### ISSUE THREAD

**Issue by piechart** [fastlane/fastlane#8043]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

```
[22:05:15]: You can tweet: "iTunes Connect #iosprocessingtime 125 minutes"
[22:05:47]: Waiting iTunes Connect processing for build 1.0 (4)... this might take a while...
[22:05:47]: 
[22:05:47]: You can tweet: "iTunes Connect #iosprocessingtime 125 minutes"
[22:06:19]: Waiting iTunes Connect processing for build 1.0 (4)... this might take a while...
```

### Environment 

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                 | Value                                       |
| ------------------- | ------------------------------------------- |
| OS                  | 10.12.2                                     |
| Ruby                | 2.4.0                                       |
| Bundler?            | false                                       |
| Git                 | git version 2.6.4                           |
| Installation Source | ~/.rvm/rubies/ruby-2.4.0/bin/fastlane       |
| Host                | Mac OS X 10.12.2 (16C68)                    |
| Ruby Lib Dir        | ~/.rvm/rubies/ruby-2.4.0/lib                |
| OpenSSL Version     | OpenSSL 1.0.2j  26 Sep 2016                 |
| Is contained        | false                                       |
| Xcode Path          | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version       | 8.2.1                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | ru_RU.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.12.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| slack-notifier            | 1.5.1        |
| i18n                      | 0.7.0        |
| json                      | 1.8.6        |
| thread_safe               | 0.3.5        |
| tzinfo                    | 1.2.2        |
| minitest                  | 5.10.1       |
| activesupport             | 4.2.7.1      |
| CFPropertyList            | 2.3.4        |
| claide                    | 1.0.1        |
| colored                   | 1.2          |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.2        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| babosa                    | 1.0.2        |
| highline                  | 1.7.8        |
| commander                 | 4.4.3        |
| excon                     | 0.54.0       |
| faraday                   | 0.11.0       |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161129 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.0.1        |
| gh_inspector              | 1.0.3        |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| hurley                    | 0.2          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.1.0        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.9.23       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.1.2        |
| bundler                   | 1.13.7       |
| faraday_middleware        | 0.11.0.1     |
| io-console                | 0.4.6        |
| openssl                   | 2.0.2        |
</details>

*generated on:* **2017-01-28**
</details>

As we know, there is an automatic check after uploading build to iTunes Connect before we can distribute this build or test via TestFlight.
It can reject build after scanning **Info.plist**, app entities or whatever.
This time I've used **submit_for_review** option in **deliver** and received an email because of missed keys in Info.plist

![36d0fbf6d5](https://cloud.githubusercontent.com/assets/17230738/22399106/098f167c-e5a7-11e6-9fb9-61c07a2ddbf7.png)

But checking by fastlane for build being proceeded has not finished. It can be easily checked because the build did not appear on iTunes Connect totally, like nothing was uploaded at all.
